core/trace/src/main/scala-2/org/typelevel/otel4s/trace/TracerMacro.scala

```diff
@@ -115,9 +115,7 @@ object TracerMacro {
       attributes: c.Expr[Attribute[_]]*
   ): c.universe.Tree = {
     import c.universe._
-    val meta = q"${c.prefix}.meta"
-
-    q"if ($meta.isEnabled) ${c.prefix}.spanBuilder($name).addAttributes(..$attributes).start else $meta.noopSpan"
+    q"${c.prefix}.spanBuilder($name).addAttributes(..$attributes)"
```

Comment:
TODO: `Meta.noopSpan` needs to return the same thing as `span`, whatever that ends up being.